tasks/build-chi-scripts.js

```diff
@@ -18,7 +18,8 @@ const webpackConfig = {
   mode: WEBPACK_MODE,
   output: {
     library: 'chi',
-    filename: 'chi.js'
+    filename: 'chi.js',
```

Comment:
Will AMD way of instantiating Chi still work as it is in the documentation?
What is the improvement?

Reply:
AMD is compiled by `build-chi-scripts-amd.js`. This change should not affect AMD.
The improvement is that currently chi global variable is declared like: `var chi = {}`. We encountered a compatibility issue with a Salesforce Lightning project which requires declarations like `window.chi = {}`.  If you look into `chi.js` compiled for this PR, you'll see the declaration change: https://ac8c80953209610b5871950fb4413029e235c068-chi-pr.enterprise.ctl.io/js/chi.js